pr_overview.py

```diff
@@ -162,24 +255,281 @@ def create_pr_overview(prs_data, gh_repo):
+                style_check = 1
+
+            if comment.lower().startswith('test report'):
+                if 'SUCCESS' in comment:
+                    test_reports = 1
+                elif 'FAILED' in comment:
+                    test_reports = -1
+
+            if 'good to go' in comment.lower():
+                signed_off = True
+
+        status = unit_test + style_check + test_reports
+
+        participants = len(nub(pr['issue_comments']['users']))
+
+        year, month, day, hour, minutes, seconds = pr['updated_at'].replace('T', '@')[:-1].replace(':', '@').replace('-', '@').split('@')
+        pr.update({
+            'age': (datetime_parser('TODAY') - datetime_parser(pr['created_at'].split('T')[0])).days,
+            'last_update': "new Date(%s, %s, %s, %s, %s, %s)" % (year, month, day, hour, minutes, seconds),
+            'participants': participants,
+            'signed_off': ['', 'true'][signed_off],
+            'status': status,
+            'style_check': TEST_RESULTS[style_check+1],
+            'test_reports': TEST_RESULTS[test_reports+1],
+            'unit_test': TEST_RESULTS[unit_test+1],
+            'user_login': pr['user']['login'],
+        })
+
+        lines.append(row_tmpl % pr)
+
+        last_update = max(datetime_parser(pr['updated_at'].replace('T', ' ')[:-1]), last_update)
+
+    lines.append('        ]);')
+
+    return len(lines)-2, '\n'.join(lines), merged_today, last_update
+
+
 def main():
+    types = {
+        'html': gen_pr_overview_page,
+        'plot': plot_pr_stats,
+        'print': pr_overview,
+    }
 
     opts = {
         'github-account': ("GitHub account where repository is located", None, 'store', 'hpcugent', 'a'),
         'github-user': ("GitHub user to use (for authenticated access)", None, 'store', 'boegel', 'u'),
         'repository': ("Repository to use", None, 'store', 'easybuild-easyconfigs', 'r'),
+        'type': ("Type of overview: 'print', html' or 'plot'", 'choice',
+                 'store_or_None', 'print', types.keys(), 't'),
     }
+
+
     go = simple_option(go_dict=opts, descr="Script to print overview of pull requests for a GitHub repository")
 
     github_token = fetch_github_token(go.options.github_user)
     github = RestClient(GITHUB_API_URL, username=go.options.github_user, token=github_token, user_agent='eb-pr-overview')
 
+    pickle_file = None
+    if go.args:
+        pickle_file = go.args[0]
+
     downloading_msg = "Downloading PR data for %s/%s repo..." % (go.options.github_account, go.options.repository)
     print(downloading_msg)
+    prs = fetch_prs_data(pickle_file, github, go.options.github_account, go.options.repository, downloading_msg)
 
-    prs_data = fetch_prs_data(github, go.options.github_account, go.options.repository, downloading_msg)
-    gh_repo = github.repos[go.options.github_account][go.options.repository]
-    create_pr_overview(prs_data, gh_repo)
+    # put options here
+    types[go.options.type](prs, go)
```

Comment:
add a check to make sure that the specified `type` is a known one, and raise an error that lists the known types if it's not (rather than throwing a nasty `KeyError` as would happen now)